### PR TITLE
os.DirEntry.stat() accepts a follow_symlinks keyword argument

### PR DESCRIPTION
**Pull request by simpkins**
This function accepts the same arguments as the is_file() and is_dir()
methods in this class.

### CODE REVIEW

File: stdlib/3/os/__init__.pyi [hunk 1]
```diff
@@ -213,7 +213,7 @@ if sys.version_info >= (3, 6):
         def is_dir(self, follow_symlinks: bool = ...) -> bool: ...
         def is_file(self, follow_symlinks: bool = ...) -> bool: ...
         def is_symlink(self) -> bool: ...
-        def stat(self) -> stat_result: ...
+        def stat(self, follow_symlinks: bool = ...) -> stat_result: ...
```
Comment (srittau) [inline stdlib/3/os/__init__.pyi]: Please mark this argument as a keyword-only argument.

```suggestion
        def stat(self, *, follow_symlinks: bool = ...) -> stat_result: ...
```

(And maybe do the same with `is_dir` and `is_file` while you are at it.)

File: stdlib/3/os/__init__.pyi [hunk 2]
```diff
@@ -227,7 +227,7 @@ elif sys.version_info >= (3, 5):
...
```
Comment (srittau) [inline stdlib/3/os/__init__.pyi]: Same here.

**Comment by simpkins** [comment]
Thanks, for the review.  Updated with the requested changes.

**Comment by srittau** [comment]
I will merge these changes when CI passed. For future reference: Please do not force commit/squash changes, just push new commits. This makes it easier to review changes. We will squash when merging into master.

**Pull request merged**